.env

```diff
@@ -1 +1,5 @@
-NODE_LOCAL_PORT=3000
\ No newline at end of file
+NODE_LOCAL_PORT=5000
+DB_PRODUCTION_URL=mongodb+srv://volunterarecoded:12345voluntera@cluster0.diiga.mongodb.net/myFirstDatabase?retryWrites=true&w=majority
+DB_DEV_URL=mongodb://127.0.0.1:27017/voluntera-dev
+PORT=8000
```

Comment:
It would be better to rename this to DEVELOPMENT_PORT. Since the other one is named PRODUCTION_PORT.